src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,77 @@
+.icon {
+  display: block;
+  width: 18px;
+  height: 16px;
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  text-decoration: none;
+  position: relative;
+
+  &--menu {
+
+    background-image: url(../images/menu.svg);
+
+    &:hover {
+      background-image: url(../images/menu_hover.svg);
+    }
+  }
+
+  &--phone {
+    margin-right: 30px;
+    background-image: url(../images/phone.svg);
+
+    @include tablet {
+      margin-right: 63px;
+    }
+
+    @include onDesktop {
+      margin-right: 63px;
+    }
+  }
+
+  &--cross {
+    background-image: url(../images/cross.svg);
+  }
+
+  @include tablet {
+    width: 24px;
+    height: 24px;
+  }
+
+  @include onDesktop {
+    width: 24px;
+    height: 24px;
+  }
+
+  &__tooltip {
+    position: absolute;
+    top: 50%;
+    transform: translate(-100%, -50%);
+    padding-right: 20px;
+    text-decoration: none;
+    margin-right: 30px;
+    color: $background-color;
+    width: max-content;
+    text-transform: uppercase;
+    text-align: end;
+    font-weight: bold;
+    @include textStandart;
+    transition: opacity 0.3s;
+
+    @include tablet {
+      opacity: 0;
+      pointer-events: none;
+    }
+
+    @include onDesktop {
+      opacity: 0;
+      pointer-events: none;
+    }
+  }
```

Comment:
idk, maybe this is the idea, but for the mobile version the hover text is not hidden :)
![The-MET](https://user-images.githubusercontent.com/25872733/188471983-2a76b2ec-c93b-4db9-b3dd-b3d3a7f2e68e.png)
 For the 320px it's a bit on the logo. I've got opacity: 0 for the mobile as well